patches/react-native+0.77.1+024+fix-surface-stopped-before-started.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/node_modules/react-native/ReactCommon/react/runtime/platform/ios/ReactCommon/RCTHost.mm b/node_modules/react-native/ReactCommon/react/runtime/platform/ios/ReactCommon/RCTHost.mm
+index b796713..5142307 100644
+--- a/node_modules/react-native/ReactCommon/react/runtime/platform/ios/ReactCommon/RCTHost.mm
++++ b/node_modules/react-native/ReactCommon/react/runtime/platform/ios/ReactCommon/RCTHost.mm
+@@ -254,7 +254,9 @@ class RCTHostHostTargetDelegate : public facebook::react::jsinspector_modern::Ho
+   surface.surfaceHandler.setDisplayMode(displayMode);
+   [self _attachSurface:surface];
+ 
+-  [_instance callFunctionOnBufferedRuntimeExecutor:[surface](facebook::jsi::Runtime &_) { [surface start]; }];
++  __weak RCTFabricSurface *weakSurface = surface;
++  // Use the BufferedRuntimeExecutor to start the surface after the main JS bundle was fully executed.
++  [_instance callFunctionOnBufferedRuntimeExecutor:[weakSurface](facebook::jsi::Runtime &_) { [weakSurface start]; }];
```

Comment:
How did it this patch end up here?

Reply:
I got the same error as described [here](https://github.com/facebook/react-native/pull/48213) and there's already a fix for this in the newest react-native versions

Reply:
And is it strictly connected to this PR?

Reply:
Yeah, I think so. I didn't get this error on other branches

Reply:
Can you please add details about this patch to the PR body? We can remove this as we bump to higher verision right?